src/Microsoft.ML.Auto/API/ExperimentSettings.cs

```diff
@@ -23,10 +51,10 @@ public class ExperimentSettings
         /// <summary>
         /// This setting controls whether or not an AutoML experiment will make use of ML.NET-provided caching.
         /// If set to true, caching will be forced on for all pipelines. If set to false, caching will be forced off.
-        /// If set to null (default value), AutoML will decide whether to enable caching for each model.
+        /// If set to <see langword="null"/> (default value), AutoML will decide whether to enable caching for each model.
         /// </summary>
         public bool? CacheBeforeTrainer = null;
```

Comment:
We had a quorum of folks decide this should be an enum: on/off/auto. Welcome to add into this PR; or we'll handle later.

Reply:
+1 -- will handle in a PR shortly w/ a few other API revs